test/eventrouter.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This is a test suite for the eventrouter
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::test::junit::declare_suite_start "test/eventrouter"
+
+espod=$( get_es_pod es )
+# check if eventrouter and fluentd with correct ViaQ plugin are deployed
+non_formatted_event_count=$( curl_es $espod /_count?q=verb:* | get_count_from_json )
```

Comment:
use `/project.*/_count` - make sure we only look at the logs from pods/containers, to avoid any strange or spurious hits